mmcv/runner/hooks/momentum_updater.py

```diff
@@ -148,6 +148,54 @@ def before_train_iter(self, runner):
                 self._set_momentum(runner, warmup_momentum)
 
 
+@HOOKS.register_module()
+class StepMomentumUpdaterHook(MomentumUpdaterHook):
+    """Step momentum scheduler with min value clipping.
+
+    Args:
+        step (int | list[int]): Step to decay the momentum. If an int value is
+            given, regard it as the decay interval. If a list is given, decay
+            momentum at these steps.
+        gamma (float, optional): Decay momentum ratio. Default: 0.5.
+        min_momentum (float, optional): Minimum momentum value to keep. If
+            momentum after decay is lower than this value, it will be clipped
+            accordingly. If None is given, we don't perform lr clipping.
+            Default: None.
+    """
+
+    def __init__(self, step, gamma=0.5, min_momentum=None, **kwargs):
+        assert isinstance(step, (list, int))
```

Comment:
seems 167 is unnecessary due to logic in 168-174.

Reply:
You are right. Also change lr_updater.